README.md

```diff
@@ -132,18 +132,20 @@ parameters: the configuration structure, v3io data container object and logger o
 you already have container and logger (when using nuclio data bindings).
 
 Configuration is specified in a YAML or JSON format, and can be read from a file using `config.GetOrLoadFromFile(path string)` 
-or can be loaded from a local buffer using `config.GetOrLoadFromData(data []byte)`. You can see details on the configuration
-options in [config](internal/pkg/config/config.go), a minimal configuration looks like: 
+or can be loaded from a local buffer using `config.GetOrLoadFromData(data []byte)`.
+You can see details on the configuration options in [config](internal/pkg/config/config.go).
```

Comment:
Path: `$GOPATH/src/github.com/v3io/v3io-tsdb/pkg/config/config.go`

Reply:
The config link target wasn't edited by me; I also don't like the text, but I didn't do a full review of the README file yet. But as you've pointed it out, I've now changed the sentence to the following:
> You can see details on the configuration options in the V3IO TSDB [**config.go**](pkg/config/config.go) source file.
Note that I removed `internal/` from the link target and left it as `pkg/config/config.go`. You can't use `$GOPATH/...` in a Mardkown link target. Using a relative file path is the best practice for such cases and works well. When reading the README file in the GH web interface, the link redirects to the relative target config file for the repo branch that you're viewing, which is what you want.